src/ray/raylet/worker_pool.h

```diff
@@ -41,6 +41,9 @@ namespace raylet {
 using WorkerCommandMap =
     std::unordered_map<Language, std::vector<std::string>, std::hash<int>>;
 
+// TODO(swang): Are we worried about hash collisions with int type?
+using RuntimeEnvHash = int;
```

Comment:
Ah I had a PR that changed it to int64 but it got lost. Maybe file a backlog issue / add to the tech debt cleanup ticket.